ezpublish/config/ezpublish.yml

```diff
@@ -0,0 +1,56 @@
+# import the ezpublish.yml from your Bundle
+imports:
+    - {resource: @EzTutorialBikeBundle/Resources/config/ezpublish.yml}
+
+ezpublish:
+    repositories:
+        demo_repository:
+            engine: legacy
+            connection: default
+
+    siteaccess:
+        # Available siteaccesses
+        list:
+            - demo_site
+            - gobike
+
+        # Siteaccess groups. Use them to group common settings.
+        groups:
+            demo_site_group: [demo_site, gobike]
+        default_siteaccess: demo_site
```

Comment:
Because of working with demo_site siteaccess : not sure that my template_rules.yml are considered.